### ISSUE TITLE
Failed to load the php-unit-integrator package

### ISSUE THREAD

**Issue by tominventisbe** [twifty/atom-phpunit-integrator#1]
Steps to reproduce:

1. Just install the package and let it activate itself.

**Atom**: 1.25.0 x64
**Electron**: 1.8.4
**Thrown From**: [php-unit-integrator](https://github.com/twifty/atom-phpunit-integrator) package 0.0.2

### Stack Trace

Failed to load the php-unit-integrator package

```
At variable @text-color-ignored is undefined in ~/.atom/packages/php-unit-integrator/styles/php-unit-integrator.less:116:17

LessError: variable @text-color-ignored is undefined
at ~/.atom/packages/php-unit-integrator/styles/php-unit-integrator.less:116:17
```

Not sure if this is something specific to my setup.

**Comment by twifty** [comment]
Which atom theme are you using? It's possible the variable names I have used have not been defined in your theme.

**Comment by twifty** [comment]
Fixed in [0.0.4](https://github.com/twifty/atom-phpunit-integrator/commit/c59185c5b132c49f7b1ee8b80d0bd48b3965fb0b)

**Comment by tominventisbe** [comment]
Was indeed fixed, thanks. It appears to have reappeared in 0.0.8:

```
'./node_modules/colletch/styles/colletch.less' wasn't found. Tried - /home/user/.atom/packages/php-unit-integrator/styles/node_modules/colletch/styles/colletch.less,/home/user/.atom/packages/nebula-syntax/styles/node_modules/colletch/styles/colletch.less,/home/user/.atom/packages/nebula-ui/styles/node_modules/colletch/styles/colletch.less,/usr/lib/atom/static/variables/node_modules/colletch/styles/colletch.less,/usr/lib/atom/static/node_modules/colletch/styles/colletch.less,node_modules/colletch/styles/colletch.less in /home/user/.atom/packages/php-unit-integrator/styles/php-unit-integrator.less:5:0

LessError: './node_modules/colletch/styles/colletch.less' wasn't found. Tried - /home/user/.atom/packages/php-unit-integrator/styles/node_modules/colletch/styles/colletch.less,/home/user/.atom/packages/nebula-syntax/styles/node_modules/colletch/styles/colletch.less,/home/user/.atom/packages/nebula-ui/styles/node_modules/colletch/styles/colletch.less,/usr/lib/atom/static/variables/node_modules/colletch/styles/colletch.less,/usr/lib/atom/static/node_modules/colletch/styles/colletch.less,node_modules/colletch/styles/colletch.less
at /home/user/.atom/packages/php-unit-integrator/styles/php-unit-integrator.less:5:0
```

**Comment by twifty** [comment]
Hmm, I missed a dot when importing the stylesheet. A big question here is why is my version of atom not giving me an error? It loads and renders without a problem even though the import is obviously wrong.

A quick fix until a push a new version:

Open `~/.atom/packages/php-unit-integrator/styles/php-unit-integrator.less` and change the line

`@import "./node_modules/colletch/styles/colletch.less";`

to 

`@import "../node_modules/colletch/styles/colletch.less";`

**Comment by twifty** [comment]
Should be fixed in 0.0.9

**Comment by tominventisbe** [comment]
Fixed indeed, thanks!
